apisix/admin/plugins.lua

```diff
@@ -85,8 +106,14 @@ function _M.get(name)
 end
 
 
-function _M.get_plugins_list()
-    local plugins = core.config.local_conf().plugins
+function _M.get_plugins_list(subsystem)
+    local plugins
+    if subsystem == "stream" then
+        plugins = core.config.local_conf().stream_plugins
+    else
+        plugins = core.config.local_conf().plugins
+    end
+
```

Comment:
The stream plugins is not in the directory `apisix.plugins`

Reply:
apisix.stream.plugins is the directory of stream plugins

Reply:
done